doc/api/fs.md

```diff
@@ -2529,6 +2529,19 @@ changes:
 Asynchronous rename(2). No arguments other than a possible exception are given
 to the completion callback.
 
+This function will take the old path name of a file and rename it to the path
```

Comment:
In this line and the line below, use _pathname_ (as seen elsewhere in the doc) and not _path name_.